app-misc/trash-cli/trash-cli-0.17.1.14_p20170816-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit distutils-r1 vcs-snapshot
```

Comment:
I wonder if you really need *vcs-snapshot* eclass here, if you define `S`.

Reply:
Yeah, prolly we can get rid of this.